core/unit_test/TestTeam.hpp

```diff
@@ -1155,15 +1155,17 @@ struct TestTeamBroadcast {
 
     value_type parUpdate = 0;
     value_type value     = tid * 3 + 1;
+    bool bChoice         = (lid % ts == tid) ? true : false;
```

Comment:
Ternary operator is unnecessary.

Reply:
Please come up with a better name than `bChoice`.  Do not encode in the name that it is boolean.  What thread(s) are you selecting?